### ISSUE TITLE
`vscode.tasks.fetchTasks()` will freeze when called in `activate` function

### ISSUE THREAD

**Issue by nhathuy13598** [microsoft/vscode#182388]
Hi team,
I'm writing an extenstion (https://github.com/ludenstian/automatask) and facing an issue that I cannot reproduce (in my lastest VSCode). When `activate` function runs, I will register a new task provider and then fetch all tasks. When I test my extesion with VSCode 1.78.2, everything work fine but when running with VSCode 1.74.3, it stucks at line where I call `await vscode.tasks.fetchTasks()`.
Does this issue occur when all extensions are disabled?: Yes, only my extension is enabled

- VS Code Version: 1.74.3
- OS Version: Windows_NT x64 10.0.19045

Steps to Reproduce:

1. Create `tasks.json` like this
```
{
    "version": "2.0.0",
    "tasks": [
        {
            "type": "shell",
            "label": "Hello World",
            "command": [
                "echo", "'Hello World'"
            ],
            "problemMatcher": []
        },
        {
            "type": "automatask",
            "filePatterns": [".*"],
            "taskToTrigger": ["Hello World"],
            "label": "Test"
        }
    ]
}
```
2. Create a dummy file
3. Press `F6` at opened dummy file to activate extesion

**Comment by meganrogge** [comment]
Already fixed in insiders

**Comment by nhathuy13598** [comment]
Hi @meganrogge, could you provide more infos about the fix as it works in version 1.78.2? Thanks
